### DISCUSSION TITLE
Laptop Power adapters

### DISCUSSION THREAD

**Discussion by BJLawsonSTM** [snipe/snipe-it#14467] [General]
I wanted some ideas on the best way to go about this. I work for a school district, and the media centers check out student devices using SnipeIT. This is the first school year they are using this system (We've used SnipeIT to track staff devices for a while and love it). They are looking for a way to check out a power adapter to the student when they check out a laptop. Students are not returning the adapters, and they need a better way to track who to charge. At first, I thought about using the predefined kit section, but they want to check out specific assets for the student based on the serial. Such as picking up a device, scanning it, checking it out using the student number, and handing it to them. Do you have any ideas on how to quickly and easily (These are librarians, not tech people) checkout a power adapter simultaneously?

**Comment by ShaunNeighbourSwale** [comment]
Have you considered checking out the power adaptor to the laptop, and the laptop to the user?  That links the two items, but only requires one user checkout?

**Reply by TShiremanLCOG** [reply]
This seems like the quickest way. If an adapter is lost or broken, you will have to check in the broken one and check out the new one. I would recommend getting a paint pen and marking the adapter with the computer number.

**Comment by mauro-miatello** [comment]
I suggest to use a internal code, for example NB23001 and PW23001 (notebook number 1 of 2023 and power adapter number 1 of 2023), and put on them your label, that is more easy to read and manage than serial number.
